fabrictestbed_extensions/fablib/node.py

```diff
@@ -2934,6 +2990,25 @@ def set_run_update_commands(self, run_update_commands: bool = True):
         self.set_fablib_data(fablib_data)
 
     def config(self, log_dir="."):
+        """
```

Comment:
Can we also mention a NOTE here that the user can use this to re-apply config post os_reboot as well?

Reply:
Done. Please take a look: https://fabric-fablib--237.org.readthedocs.build/en/237/node.html#fabrictestbed_extensions.fablib.node.Node.config

Reply:
Looks good!